### PR TITLE
Update to latest discussion

### PR DESCRIPTION
**Pull request by philpax**
Reflecting recent developments from discussion and implementation

### CODE REVIEW

File: text/17-automated-build-and-submit-pipeline.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ Each folder contains files that correspond to a submitted plugin, which is what
 [plugin]
 # The open-source Git repository that your code is hosted on.
 # Must be visible to the external web, but doesn't have to be GitHub.
+# If `git clone` accepts it, it will work here.
```
Comment (NotNite) [inline text/17-automated-build-and-submit-pipeline.md:43]: We should specify HTTP remotes - we won't have an SSH key, so SSH will fail.

**Comment by Dormanil** [comment]
LGTM

**Review by NotNite — APPROVED** [review]
(I probably don't have the proper authorization to approve this)

**Comment by philpax** [comment]
> (I probably don't have the proper authorization to approve this)

You're the second most involved in the implementation right now, it's fine if you approve it

**Pull request merged**